birdhouse/env.local.example

```diff
@@ -206,17 +208,24 @@ export POSTGRES_MAGPIE_PASSWORD=postgres-qwerty
 #
 #export ALLOW_UNSECURE_HTTP=""
 
-# Jupyter single-user server images
+# Jupyter single-user server images=
 #export DOCKER_NOTEBOOK_IMAGES="pavics/workflow-tests:210216 \
 #                               pavics/crim-jupyter-eo:0.1.0 \
 #                               pavics/crim-jupyter-nlp:0.1.0"
 
+# Name of the images displayed on the JupyterHub image selection page
+# The name order must correspond to the order of the DOCKER_NOTEBOOK_IMAGES variable,
+# and both variables should have the same number of entries.
+#export JUPYTERHUB_IMAGE_SELECTION_NAMES="pavics \
+#                                         eo-crim \
+#                                         nlp-crim"
+
 # allow jupyterhub user selection of which notebook image to run
 # see https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html
 #export ENABLE_JUPYTERHUB_MULTI_NOTEBOOKS="
-#c.DockerSpawner.image_whitelist = {'pavics': os.environ['DOCKER_NOTEBOOK_IMAGES'].split()[0],
-#                                   'eo-crim': os.environ['DOCKER_NOTEBOOK_IMAGES'].split()[1],
-#                                   'nlp-crim': os.environ['DOCKER_NOTEBOOK_IMAGES'].split()[2],
+#c.DockerSpawner.image_whitelist = {os.environ['JUPYTERHUB_IMAGE_SELECTION_NAMES'].split()[0]: os.environ['DOCKER_NOTEBOOK_IMAGES'].split()[0],
+#                                   os.environ['JUPYTERHUB_IMAGE_SELECTION_NAMES'].split()[1]: os.environ['DOCKER_NOTEBOOK_IMAGES'].split()[1],
+#                                   os.environ['JUPYTERHUB_IMAGE_SELECTION_NAMES'].split()[2]: os.environ['DOCKER_NOTEBOOK_IMAGES'].split()[2],
```

Comment:
For these lines, I am aware it is not super clean. I haven't found an easier way to implement it yet, since the concerned variables have to be working both on shellscript and python.

About `DOCKER_NOTEBOOK_IMAGES` and `JUPYTERHUB_IMAGE_SELECTION_NAMES`, we could have use those names in just one variable instead of 2, using a string "formatted similarly to a dictionary" such as `"pavics;pavics/workflow-tests:210216 eo-crim;pavics/crim-jupyter-eo:0.1.0 ...."` but it would require additional string splitting, so I don't think it would be cleaner.

Another thing is we could probably simplify this by using the same name all the time, instead of having 2 different name format. We would use the image name such as `pavics/crim-jupyter-eo:0.1.0` everywhere though, meaning it would be what we see in the jupyterhub list, and it would be the name of the directories for the tutorial-notebooks, instead of the shorter version `eo-crim`. I don't think it would be necessarily cleaner either. (Potential problem with the '/' found in the image names, clashing with using it as a directory name?)

Reply:
Agreed it's ugly, but also agree other alternative either complexify the implemenation a lot or look less great.  Open to other alternative as well, if someone see something we both missed ...